packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -160,16 +130,13 @@ export function EntitiesSavedStatesExtensible( {
 			</Flex>
 
 			<div className="entities-saved-states__text-prompt">
-				<div
-					className="entities-saved-states__text-prompt--header-wrapper"
-					id={ renderDialog ? dialogLabel : undefined }
-				>
+				<div className="entities-saved-states__text-prompt--header-wrapper">
 					<strong className="entities-saved-states__text-prompt--header">
 						{ __( 'Are you ready to save?' ) }
 					</strong>
 					{ additionalPrompt }
 				</div>
-				<p id={ renderDialog ? dialogDescription : undefined }>
```

Comment:
removing this id will make the dialog aria-describedby fail

Reply:
Do you think passing ids via props will make any sense to keep component clean? I don't think current approach can handle adding them otherwise.